js/src/components/google-combo-account-card/connect-ads/connect-cta.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppButton from '.~/components/app-button';
+import ConnectedIconLabel from '.~/components/connected-icon-label';
+import useEventPropertiesFilter from '.~/hooks/useEventPropertiesFilter';
+import { FILTER_ONBOARDING } from '.~/utils/tracks';
+
+/**
+ * Connect CTA component.
+ *
+ * @param {Object} props Props.
+ * @param {boolean} props.isConnected Whether the account is connected.
+ * @param {number} props.id Account ID.
+ * @param {Function} props.handleConnectClick Callback to handle the connect click.
+ * @param {string} props.value Connected account ID.
+ *
+ * @return {JSX.Element} Connect CTA component.
+ */
+const ConnectCTA = ( { isConnected, id, handleConnectClick, value } ) => {
```

Comment:
@ankitrox We don't need the `id` prop since `value` should be enough to connect to a new account.